### ISSUE TITLE
Inconsistency in operator mode hops.

### ISSUE THREAD

**Issue by Neelfrost** [phaazon/hop.nvim#288]
### Current behavior/Problem:

Currently, operator (say e.g., yanking) mode hops are not consistent with normal hops.

```
vim.keymap.set("n", "f", "<cmd>HopChar1<CR>")
vim.keymap.set("o", "f", function()
    require("hop").hint_char1({
        direction = require("hop.hint").HintDirection.AFTER_CURSOR,
        current_line_only = true,
        -- hint_offset = 1, -- require("hop.hint").HintDire
        -- hint_offset = 0, -- require("hop.hint").HintDi
        -- hint_offset = -1, -- require("hop.hint").HintDi
    })
end)
```

Text used for example hop: require("hop.hint").HintDirection.AFTER_CURSOR (from begining to the 'r' in 'HintDirection').

In normal mode, hop will bring the cursor to 'r', but in operator mode, its always either after 'r' or before as can be seen in the comments next to the hint_offsets.

### Required behavior:

Basically, `hint_offset = 0` should operate upto 'r' i.e., require("hop.hint").HintDir to be consistent with the normal mode behavior.

**Comment by phaazon** [comment]
That should be fixed, try again with `v2.0.2` (or just update if you are using `v2`, `v2.0` or `master`).

**Comment by Neelfrost** [comment]
Works as intended, Thank You! :)
